CITATION.cff

```diff
@@ -0,0 +1,9 @@
+cff-version: 1.2.0
+message: "To cite DART, please use the following metadata. Update the DART version and year as appropriate."
+title: "The Data Assimilation Research Testbed"
+version: "X.Y.Z"
+date-released: "2024-03-13"
+doi: "http://doi.org/10.5065/D6WQ0202"
```

Comment:
```suggestion
doi: "10.5065/D6WQ0202"
```

The doi is coming out as 
UCAR/NCAR/CISL/DAReS. (2024). The Data Assimilation Research Testbed (Version X.Y.Z) [Computer software]. https://doi.org/http://doi.org/10.5065/D6WQ0202

on the APA citation.